apps/ltn/src/impact/ui.rs

```diff
@@ -422,26 +425,7 @@ fn export_csv(app: &App) -> Result<String> {
     {
         let mut writer = csv::Writer::from_writer(&mut out);
         for r in app.per_map.map.all_roads() {
-            writer.serialize(ExportRow {
-                road_name: r.get_name(None),
-                osm_way_id: r.orig_id.osm_way_id.0,
-                osm_intersection1: r.orig_id.i1.0,
-                osm_intersection2: r.orig_id.i2.0,
-                total_count_before: app
-                    .per_map
-                    .impact
-                    .compare_counts
-                    .counts_a
-                    .per_road
-                    .get(r.id),
-                total_count_after: app
-                    .per_map
-                    .impact
-                    .compare_counts
-                    .counts_b
-                    .per_road
-                    .get(r.id),
-            })?;
+            writer.serialize(ExportRow::new(r, app))?
```

Comment:
```suggestion
            writer.serialize(ExportRow::new(r, app))?;
```
The semicolon missing weird me out a bit O_O